controllers/cron.ctrl.php

```diff
@@ -25,11 +25,16 @@ class CronController extends Controller {
     
 	var $cronList;			    // the array includes all tools avialable for cron
 	var $repTools;			    // the array includes all tools avialable for report generation
-	var $debug = true;		    // to show debug message or not
+	var $debug = false;		    // to show debug message or not
 	var $layout = 'ajax';       // ajax layout or not
 	var $timeStamp;             // timestamp for storing reports
 	var $checkedKeywords = 0;   // the number of keywords checked in cron, this is used for split cron execution feature       	
 	
+	# initialize preperties

```

Comment:
Done. Thanks for the notification.



Regards,
Geo Varghese
www.seofreetools.net

On Wed, Oct 21, 2015 at 6:00 PM, Viktor Szépe <notifications@github.com>
wrote:

> In controllers/cron.ctrl.php
> <https://github.com/seopanel/Seo-Panel/pull/6#discussion_r42616159>:
>
> >  	var $layout = 'ajax';       // ajax layout or not
> >  	var $timeStamp;             // timestamp for storing reports
> >  	var $checkedKeywords = 0;   // the number of keywords checked in cron, this is used for split cron execution feature       	
> >  	
> > +	# initialize preperties
>
> A typo here!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/seopanel/Seo-Panel/pull/6/files#r42616159>.
>